main.go

```diff
@@ -33,20 +52,25 @@ func main() {
 	flag.Parse()
 
 	if *versionFlag {
-		_, _ = fmt.Fprintf(os.Stdout, "Version: %s\n", appVersion)
-		return
+		_, _ = fmt.Fprintf(out, "Version: %s\n", appVersion)
+		return 0
 	}
 
 	cfg, err := configs.Read(configFilePath)
 	if err != nil {
-		log.Fatal().Err(err).Msg("Failed to read config")
+		log.Err(err).Msg("Failed to read config")
```

Comment:
Will logging an error automatically make it use error logging level (`log.Error()...`). Noted that you just removed `.Fatal()` without replacing it with another level